adobe/aem-core-cif-components
# CIF-1062 Implement the account creation process

## Description

This feature will allow visitors to create an account. The back-end interaction is handled by the `UserContext` and the user input is handled by the `CreateAccount` component.

Steps for verification:
1. Go to the Sign In form
2. Click the "Create Account" button
3. Fill the form with the required information - first name, last name, email, password, password confirmation
4. Click `Submit`

The user is immediately logged in after the account is created and the "My Account" section is shown. Check that the information matches the one entered in the account creation form.

Sign out and repeat the process. Check that the "My account" information is consistent with the new information entered in the "Create Account" form.

## Related Issue
CIF-1062
## Motivation and Context



## How Has This Been Tested?
Functional testing and unit testing
## Screenshots (if appropriate):

## Types of changes




## Checklist:
